source/table-filters/apply-table-filters.txt

```diff
@@ -45,6 +45,9 @@ Steps
    radio button and enter a value. This applies a limit clause at the 
    end of your SQL query.
 #. Click :guilabel:`Save and close`.
+   Tables that have applied table filters display a :icon-fa5:`filter`
```

Comment:
[suggestion] I think we can slightly reword this to avoid the confusion with the [participle](https://www.mongodb.com/docs/meta/style-guide/writing/clarify-gerunds-participles/#clarify-gerunds-and-participles), adopt more of an [active voice](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/#use-active-voice), and flow better from the preceding sentence. 

Maybe something along the lines of "After you save your filter, the :guilabel:`Schema model` pane displays a :icon-fa5:`filter` icon under the  :guilabel:`Relational` header."